include/deal.II/dofs/dof_tools.h

```diff
@@ -1382,6 +1382,30 @@ namespace DoFTools
                                const DoFHandler<dim, spacedim> &dof_handler,
                                const ComponentMask             &component_mask,
                                std::vector<std::vector<bool>>  &constant_modes);
+
+  /**
+   * Same as the above but additional to the translational modes also
+   * the rotational modes are added, needed to set up AMG for solving
+   * elasticity problems.
+   */
+  template <int dim, int spacedim>
+  void
+  extract_elasticity_modes(const Mapping<dim, spacedim>     &mapping,
```

Comment:
The natural way to call these in the mechanical engineering community is actually `rigid_body_modes`. Would you mind renaming this to `rigid_body_modes`? It is actually not only useful for elasticity problems, but for any translation and rotation invariant equation.

Reply:
I agree. Let's call it `rigid_body_modes`.